uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Hook/PageRun/AddLeaveAndAbsencesVar.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class CRM_HRLeaveAndAbsences_Hook_PageRun_AddLeaveAndAbsencesVar {
```

Comment:
For the class name, it's usually better to go for a noun instead of verbs. I'd suggest to change `AddLeaveAndAbsencesVar` to something like `LeaveAndAbsencesVarsAdder`